cdc/tests/integration_tests/flow_control/run.sh

```diff
@@ -30,6 +32,7 @@ EOF
 	run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY --config $WORK_DIR/tikv-cdc-config.toml
 	rss0=$(ps -aux | grep 'tikv-cdc' | head -n1 | awk '{print $6}')
 	if [[ $rss0 == "" ]]; then
+		echo "Failed to get rrs by ps"
```

Comment:
```suggestion
		echo "Failed to get rrs0 by ps"
```
To distinguish failure from `rss0` or `rss1`.

Reply:
Modified, thanks!